src/components/dashboard/components/EventCard.jsx

```diff
@@ -0,0 +1,32 @@
+import { Button } from '@nextui-org/react'
+import Image from 'next/image'
+import React from 'react'
+
+export const EventCard = ({ id, image, name, description, date, location, participants }) => {
+  participants = participants.slice(0, 3)
```

Comment:
This is bad practice in React.
But it's fine. what i suggest is to create new variable instead of modifiying the props. 😄

Reply:
noted mas. thank you 👍